.devcontainer/advanced-integration/welcome-message.txt

```diff
@@ -0,0 +1,5 @@
+👋 Welcome to the "Checkout Advanced Integration Example" in GitHub Codespaces!
+
+🛠️  Your environment is fully setup with all the required software.
```

Comment:
nit: extra space here

Reply:
I know it sounds strange but the extra spacing is needed to be able to render the msg properly 😅 

With space
![image](https://user-images.githubusercontent.com/24955913/234707543-919e885c-f133-44fb-9a29-8191b440499a.png)

Without 

![image](https://user-images.githubusercontent.com/24955913/234707686-47aef789-6a65-460e-aba2-4e82f117afa9.png)

I'll see if I could use some other emoji and avoid this